spec/integration_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+require 'mumukit/bridge'
+
+describe 'runner' do
+  let(:bridge) { Mumukit::Bridge::Bridge.new('http://localhost:4567') }
+
+  before(:all) do
+    @pid = Process.spawn 'RACK_ENV=development rackup -p 4567'
+    sleep 3
+  end
+  after(:all) { Process.kill 'QUIT', @pid }
+
+  context 'when submission is ok' do
+    it 'answers a valid hash' do
```

Comment:
It is an integration test that validate that test runner, test compiler and expectation runner properly work together.